ui/partials/perf/comparerror.html

```diff
@@ -0,0 +1,2 @@
+<div class="alert alert-danger" role="alert" ng-repeat="error in errors"><span class="fa fa-warning"></span> {{error}}</div>
+<a href="#/comparechooser?originalProject={{originalProject}}&originalRevision={{originalRevision}}&newProject={{newProject}}&newRevision={{newRevision}}">Modify choice</a>
```

Comment:
Hm, well what I was thinking is the monospace font would only be applied to the SHA, not the entire line (which would remain it's former font family, Helvetica Neue, Arial, etc).

But what is also interesting is the top sha is 10 characters, and the bottom sha is 11 characters. That is another reason for the line length discrepancy. I think Treeherder uses a filter to always present its SHA's with 12 characters.

Unrelated, just wondering is there a reason why the error is presented in a paragraph tag? In case there's benefit to just use a div of some kind so the error is just the dimensions of the message. But whatever you guys prefer is fine w/me :)